python/cudf_polars/cudf_polars/dsl/expressions/sorting.py

```diff
@@ -43,6 +43,9 @@ def do_evaluate(
     ) -> Column:
         """Evaluate this expression given a dataframe for context."""
         (child,) = self.children
+        name = None
+        if isinstance(child, Col):
```

Comment:
Curious, can `child` not be a `Col` here?

Reply:
Yes, I think it can be another expression like a literal or a binary operation inside a group_by.